### ISSUE TITLE
To-do List

### ISSUE THREAD

**Issue by ajaxvp** [superischroma/asc#1]
The following is a collection of objectives for the development team behind A#'s compiler to reach as time goes on.

**High priority:** These will be fulfilled first.

- [x] Create while loops
- [x] Fix variable assignment to itself
- [ ] Support for pointers
- [ ] Support for more operators (*, /, ==, *, &, etc.)
- [ ] Add floating point variable types
- [ ] Add multiplication, division, and modulus operations
- [ ] Create standard library which contains useful functions to be used in programs
- [ ] Create "types" (essentially C structures)
- [ ] Fix bugs with nested blocks (ifs inside of whiles, etc.)
- [ ] Constants

**Low priority:** These will be fulfilled once the project has reached a state at which they can be implemented.

- [ ] Add object orientation
- [ ] Basic inheritance with object orientation
- [x] Drop support for hardcodes <s>once standard library is ready</s>

**Comment by ajaxvp** [comment]
Overridden by #2.
